one/src/main.rs

```diff
@@ -220,6 +220,40 @@ impl Daemon {
         };
         debug!("using directory: {}", dir.display());
 
+        let mut bootstrap_addresses = opts.bootstrap_addresses.clone();
```

Comment:
I believe we decided to remove the `bootstrap_addresses` from the CLI options. Can you make that change?

Reply:
removed